pandas/core/common.py

```diff
@@ -55,7 +55,11 @@ def flatten(l):
 def _consensus_name_attr(objs):
     name = objs[0].name
     for obj in objs[1:]:
-        if obj.name != name:
+        name_check = (obj.name != name)
+        if not isinstance(name_check, (bool, np.bool_)):
```

Comment:
Can you use a try...except block here to catch the ValueError instead of type introspection?

Reply:
@WillAyd thanks - will make the changes. Should I add whatsnew note to v0.23.1?

Reply:
Yes that works

Reply:
Done